CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+##
+
+Breaking: Rename `auth.jwt.generateToken` to `auth.jwt.generateServerSignedToken` and `auth.jwt.verifyToken` to `auth.jwt.verifyServerSignedToken`.
```

Comment:
The changelog is not used for a long time, because it's very tedious.
We use GitHub releases to indicate breaking changes.
The breaking changes should be indicated in your PR description, and we'll add a warning in the release.

Reply:
Should we delete the file all together then?

Reply:
Yes probably.